src/rockstor/storageadmin/views/rockon_helpers.py

```diff
@@ -241,35 +242,47 @@ def generic_install(rockon):
         cmd.extend(port_ops(c))
         cmd.extend(container_ops(c))
         cmd.extend(envars(c))
-        cmd.append(c.dimage.name)
+        cmd.append(image_name_plus_tag)
         cmd.extend(cargs(c))
-        run_command(cmd)
+        run_command(cmd, log=True)
 
 
 def openvpn_install(rockon):
+    """
+    Custom config for the openvpn Rock-on install.
+    :param rockon:
+    :return:
+    """
     # volume container
     vol_co = DContainer.objects.get(rockon=rockon, launch_order=1)
     volc_cmd = list(DCMD) + ['--name', vol_co.name, ]
     volc_cmd.extend(container_ops(vol_co))
-    volc_cmd.append(vol_co.dimage.name)
-    run_command(volc_cmd)
+    image_name_plus_tag = vol_co.dimage.name + ':' + vol_co.dimage.tag
```

Comment:
My only concern is that this function does not seem to be in use anymore. It doesn't hurt to also implement your changes there, of course.

Reply:
@FroggyFlox Yes I thought this myself so during development I changed their names and then build from scratch. Should have indicated this, partly why I put the docstrings in. If you build with these missing/removed/renamed then the post install instructions in the more_info of for example the OpenVPN Rock-on fails. Also note the command executions that only exist in these as I did logging checks as well. So yes it would be nice to indicate where they are run from and pop that in the docstrings as well. Sorry but I just moved on and didn't follow through.

Reply:
My check for the owncloud_install was to turn on log for the db specific command and then install, you can then see the db stuff logged. Apologies for not noting it's caller, and PyCharm didn't find it either. But they are both called upon the specific rock-ons being executed. We will find it and must then mark otherwise this will haunt us again. But not necessarily in this pr :).

Reply:
Thanks for the information, these have been bothering me for quite a while and I can't seem to find their trigger anywhere. I'll follow the same route you did and hopefully will find where they're coming from.
Thanks again!